.gitignore

```diff
@@ -5,4 +5,5 @@
 *.sublime-workspace
 
 # Custom
-coverage.txt
\ No newline at end of file
+coverage.txt
+tmp
```

Comment:
Why we need it?

Reply:
Running `go test -v .` at the root dir would generate a folder tmp for trust store due to the example_test files.

Reply:
Shouldn't we be cleaning that file after test run?